src/layouts/NativeLayout.vue

```diff
@@ -259,3 +259,9 @@ export default {
   background: #8946df;
 }
 </style>
+
+<style lang="scss">
+body .c-native-layout {
+    color: white;
```

Comment:
should this be a variable or straight up white?

Reply:
In this case it's just white. This is for native which doesn't have any variables or anything set up. On the evm side we use variables because we are future proofing for dark mode